tests/create_basic_alarm/main.tf

```diff
@@ -1,22 +1,60 @@
 locals {
-  metric_alarms = [
+  metric_name_alarms = [
     {
       alarm_name                = "terraform-metric-alarm-test"
       comparison_operator       = "GreaterThanOrEqualToThreshold"
-      evaluation_periods        = "2"
+      evaluation_periods        = 2
       metric_name               = "CPUUtilization"
       namespace                 = "AWS/EC2"
-      period                    = "120"
+      period                    = 120
       statistic                 = "Average"
-      threshold                 = "80"
+      threshold                 = 90
       alarm_description         = "This metric monitors ec2 cpu utilization"
       insufficient_data_actions = []
     }
   ]
+
+  metric_query_alarms = [
+    {
+      alarm_name                = "terraform-metric-alarm-test"
+      comparison_operator       = "GreaterThanOrEqualToThreshold"
+      evaluation_periods        = 2
+      metric_name               = null
+      namespace                 = null
+      period                    = null
+      statistic                 = null
```

Comment:
I think the logic is in place such that there is no need to specify `metric_name`, `namespace`, `period`, or `statistic`, right? The user isn't _required_ to set these to null? I think it would be more clear to remove these argument, in that case...